src/pages/Follow/components/FollowProfile.tsx

```diff
@@ -86,10 +103,11 @@ export default function FollowProfile() {
           {data && data.content.map(
             (item) => (
               <FollowReview
```

Comment:
```suggestion
          {data && data.content.map((item) => (
            <FollowReview
              key={item.id}
              placeId={item.placeId}
              name={item.name}
              category={item.category}
              followerId={id}
            />
          ))}
```

여기도 줄일 수 있어요! 저는 이런 것만 보나 봐요ㅎ..